paddle/fluid/framework/details/multi_devices_graph_pass.cc

```diff
@@ -203,17 +211,19 @@ std::unique_ptr<ir::Graph> MultiDevSSAGraphBuilderBase::ApplyImpl(
 
           for (size_t i = 0; i < backward_vars.size(); i += 2) {
             auto &p_name = backward_vars[i];
-            auto &g_name = backward_vars[i + 1];
-            VLOG(10) << "Bcast " << g_name << " for parameter " << p_name;
+            auto &grad_name = backward_vars[i + 1];
+            VLOG(10) << "Bcast " << grad_name << " for parameter " << p_name
+                     << " op_type " << node->Op()->Type();
 
-            InsertCollectiveOp(&result, p_name, g_name);
+            InsertCollectiveOp(&result, p_name, grad_name);
```

Comment:
This did change nothing, can move back to original lines

Reply:
Done.Thanks!